misc/xml-to-record-converter/src/test/java/io/ballerina/xmltorecordconverter/XMLToRecordConverterTests.java

```diff
@@ -397,6 +402,15 @@ public void testXMLWithoutNamespaces() throws IOException {
         Assert.assertEquals(generatedCodeBlock, expectedCodeBlock);
     }
 
+    @Test(description = "testXMLWithMultipleNamespacesAndSameElement")
+    public void testXMLWithMultipleNamespacesAndSameElement() throws IOException {
+        String xmlFileContent = Files.readString(sample26XML);
+        String generatedCodeBlock = XMLToRecordConverter.convert(xmlFileContent, false, false, false,
```

Comment:
For the same example, shall we check the skipNamespace case as well?

Reply:
Added more tests